codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/IntermediateModel.java

```diff
@@ -52,6 +53,9 @@
     @JsonIgnore
     private final Map<String, WaiterDefinitionModel> waiters;
 
+    @JsonIgnore
+    private final Map<String, PaginatorDefinition> paginators;
```

Comment:
Can we keep consistent naming? For waiters, its WaiterDefinitionModel.

Reply:
Waiters are removed. Keep it as PaginatorDefinition.